handler.py

```diff
@@ -42,7 +42,7 @@ def default(self, o):
         return super(DecimalEncoder, self).default(o)
 
 def removeProjectFromCalendarEvents(list_of_event_ids):
-    calendarURL = "https://calendar.photonranch.org/dev/remove-project-from-events"
+    calendarURL = f"https://calendar.photonranch.org/{stage}/remove-project-from-events"
```

Comment:
The production calendar URL is `...org/calendar/...`, not `...org/prod/...` so we'll have to add an extra check before injecting the stage here.